client/source/js/app.js

```diff
@@ -85,6 +85,16 @@ define([
     })
 
     .run(function ($rootScope, $state, UserManager, localStorage, activeProject) {
+
+      if (navigator.userAgent.indexOf('MSIE') !== -1 || navigator.appVersion.indexOf('Trident/') !== -1) {
+          modalService.inform(
```

Comment:
@kitze you didn't inject the modalService into the run function so it won't work. Can you please provide a jasime test for it?